components/tidb_query/src/expr/scalar_function.rs

```diff
@@ -763,7 +765,10 @@ dispatch_call! {
         LogicalOr => logical_or,
         LogicalXor => logical_xor,
 
-        UnaryNot => unary_not,
+        UnaryNotInt => unary_not,
+        UnaryNotDecimal => unary_not,
```

Comment:
this will cause panics

Reply:
Or add a `FIXME` comment before `/*L769-L770*/`.

Reply:
Added a FIXME